tests/test_tfrecord_dataset.py

```diff
@@ -432,33 +509,32 @@ def _dict_to_tuple(sample):
         }
 
     # load dataset.
-    dataset = create_dataset(
-        setup.fname_pattern,
-        batch_size,
-        setup.parser.parse,
-        patch_size=None,
-        shuffle_buffer=5,
-        drop_remainder=drop_remainder,
-        transforms=[_dict_to_tuple],
-        cache_after_parse=False)
+    dataset = create_dataset(setup.fname_pattern,
+                             batch_size,
+                             setup.parser.parse,
+                             patch_size=None,
+                             shuffle_buffer=5,
+                             drop_remainder=drop_remainder,
+                             transforms=[_dict_to_tuple],
+                             cache_after_parse=False)
 
     # create model and train
     def _construct_model():
         '''create model with two outputs.
 
         '''
 
-        first_input = tf.keras.layers.Input(
-            shape=(None, None, 1), name='image')
-        x = tf.keras.layers.Conv2D(
-            4, kernel_size=3, padding='same')(first_input)
+        first_input = tf.keras.layers.Input(shape=(None, None, 1),
+                                            name='image')
+        x = tf.keras.layers.Conv2D(4, kernel_size=3,
+                                   padding='same')(first_input)
```

Comment:
This formatting looks inconsistent to me. What code formatter do you use?

Reply:
yapf, it's setup as a commit hook. Tried autopep8 previously but it was even worse. e.g. run it twice and get different results.